### ISSUE TITLE
Does not support terraform 0.14.2

### ISSUE THREAD

**Issue by wilanc** [mineiros-io/terraform-github-repository#55]
Tried to use module on Azure Cloud Shell:
```
Initializing modules...
Downloading mineiros-io/repository/github 0.5.1 for repository...
- repository in .terraform/modules/repository

Error: Unsupported Terraform Core version

  on .terraform/modules/repository/versions.tf line 6, in terraform:
   6:   required_version = ">= 0.12.20, < 0.14"
```

**Comment by mariux** [comment]
Thanks for catching this. will issue a release supporting TF 0.14 asap.

**Comment by mariux** [comment]
we just released v0.6.0 with support for terraform v0.14.x.. happy to get feedback if everything is working as expected.

merry Xmas ;) :santa:
